README.md

```diff
@@ -1,81 +1,92 @@
+See the [Have I Been Pwned about page](https://haveibeenpwned.com/About) for
+the "what" and "why" of data breach alerts.
 
-This project aims to explore how we, as the user agent, can attempt to do this. We're teaming up with https://haveibeenpwned.com/ [1][2] as our data source.
+This project aims to explore how Firefox - as the user agent - can support this. 
 
-[1] https://haveibeenpwned.com/About
-[2] https://en.wikipedia.org/wiki/Have_I_Been_Pwned%3F
+See [the Breach Alert Product
+Brief](https://docs.google.com/document/d/1GTS0HIihfTErA7P19HPYfvHCA3v9g67B_Cf2bpmE0Bw/edit) for more background, objectives, key use
+cases.
 
-## Basic Goals
+## Development
+### Requirements
 
-1. Inform users about data breaches through the Firefox UI - for example, a notification when they visit a site (or maybe when they focus a form on a login page) known to have recently been breached.
-2. Expose documentation/educational information about data breaches in the Firefox UI - for example, a "Learn more" link in the notification mentioned above leading to a support page.
-3. Offer a way for interested users to learn about and opt into a service that notifies them (e.g. via email) when they may be affected by breaches in the future.
+* [node](https://nodejs.org/) (with npm)
 
-## Privacy Concerns
+### Install
 
-The third goal brings up some privacy concerns, since users would need to supply an email address to receive notifications. Who is the custodian of this data? Can we avoid sending user data to haveibeenpwned.com? Can we still offer useful functionality to users who opt out of subscribing their email address? While the project is still in infancy, the idea is to offer as much utility as possible while respecting the user's privacy.
+1. clone and change to the directory:
 
-## Try it (basic testing instructions for Unix OSes as of Nov 23 2017)
+        git clone https://github.com/nhnt11/blurts-server.git
+        cd blurts-server
 
-### Clone
+2. Install dependencies:
 
-```
-$ git clone https://github.com/nhnt11/BreachAlerts.git
-```
+        npm install
 
-### Server (to test subscribing an email address and sending a "Breach Alert" email)
+3. Copy the `.env-dist` file to `.env`:
 
-Prerequisites: Node.js and npm (example commands assume you have `node` and `npm` in your path)
+        cp .env-dist .env
 
-You'll need to supply SMTP credentials for sending the email. To do so, create a file `smtp-credentials.json` in the `BreachAlerts/server` folder:
+### Run
 
-```
-{
-  "username": "<username>",
-  "password": "<password>"
-}
+1. Source the `.env` file to set required environment variables:
 
-```
+        source .env
+
+2. Run the server:
 
-NOTE: Currently the Gmail SMTP server is hardcoded, so you'll need to modify that in server.js if you're using something else.
+        node server.js
```

Comment:
npm start is the preferred way to run the server, I'd say. The "start" script is defined (to run node server.js) in package.json, and if we want to have a more sophisticated run script we can just modify that without changing the instructions.